README.md

```diff
@@ -109,6 +109,7 @@ Output is encoded as JSON and characterized by either `null`
 or a dictionary with any of the following keys:
 
 > `bundleIdentifier`
+`parentAppBundleIdentifier`
```

Comment:
I'd rename this to "parentApplicationBundleIdentifier" to stay true to the original key names.